src/views/previous/address.njk

```diff
@@ -10,9 +10,9 @@
 {% block mainContent %}
 
     {% if values.checkDetailsHeader === true %}
-      <h1 id="header" class="govuk-heading-l">{{translate("pages.address-form.previous.check-details.title")}}</h1>
+      <h1 id="header" class="govuk-heading-l">{{translate("pages.address-form-previous.check-details.title")}}</h1>
```

Comment:
Ah nice spot

Reply:
Can't we just have

```
<h1 id="header" class="govuk-heading-l">{{translate(values.addressFormTitle)}}</h1>
```
values.addressFormTitle = "pages.address-form.previous.check-details.title";

I don't know if you'll have to changing a lot of places

I just don't really fancy conditionals in the template, maybe a future refactoring work

Reply:
Agree that would be nicer, slightly tricky due to them sharing controllers. If i could get these changes through first please, I can look at a separate PR for this refactor.